libs/platform/experience/composition/src/experience-composition.component.ts

```diff
@@ -27,13 +35,13 @@ export class ExperienceCompositionComponent extends ContentMixin<CompositionProp
 ) {
   static styles = [compositionStyles];
 
-  @property()
+  @property({ reflect: true })
```

Comment:
What about making both uid and route optional (undefined) by default?

Reply:
I can. But it will be the same behavior. But instead of empty string we will emit undefined